test/cookbooks/osl-imap-test/recipes/no_ssl.rb

```diff
@@ -0,0 +1,11 @@
+include_recipe 'osl-imap-test::default'
+
+osl_imap_dovecot 'default' do
+  auth_type :system
+  ssl_cert nil
+  ssl_key nil
```

Comment:
I believe these should be nil by default. Can you please test this without setting them nil? Also, can you add a test suite to kitchen and some integration tests to ensure this works properly?